x/avs/types/keys.go

```diff
@@ -15,17 +15,10 @@ const (
 
 	// RouterKey defines the module's message routing key
 	RouterKey = ModuleName
-
-	// MemStoreKey defines the in-memory store key
-	MemStoreKey = "mem_avs"
 )
 
 const (
-	// prefixAVSInfo is the prefix for the AVS info store. It starts with a value of 5
-	// for backward compatibility with the previous prefix used for the AVS info store.
-	// TODO at the time of a chain-id upgrade, this may be reset to 1.
 	prefixAVSInfo = iota + 5
-	prefixAVSOperatorInfo
```

Comment:
Please restore this key prefix since otherwise this change is backwards incompatible

You can remove it when there is an upgrade

Reply:
fixed